src/ResourceManager/DataMigration/Commands.DataMigration.Test/Commands.DataMigration.Test.csproj

```diff
@@ -0,0 +1,227 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
@sichoudh for your tests to run during CI, you will need to add a mapping to your test .dll in [this file](https://github.com/Azure/azure-powershell/blob/preview/TestMappings.json)

You can use any of the existing mappings as an example.

Reply:
Done